tests/features/site_upstream_updates.feature

```diff
@@ -5,7 +5,7 @@ Feature: site upstream-updates
     Given I am authenticated
     And a site named "[[test_site_name]]"
     And the connection mode of "[[test_site_name]]" is "git"
-    When I run "terminus site upstream-updates --site=[[test_site_name]]"
+    When I run "terminus site upstream-updates show --site=[[test_site_name]]"
```

Comment:
I think that we should be using "list" for the sake of uniformity among the commands.